pkg/api/controller_test.go

```diff
@@ -924,9 +925,12 @@ func TestController_DeleteRepositoryHandler(t *testing.T) {
 		resp, err := clt.DeleteRepositoryWithResponse(ctx, "my-new-repo")
 		verifyResponseOK(t, resp, err)
 
+		// wait for cache expiry
+		time.Sleep(ref.RepositoryCacheExpiry + ref.RepositoryCacheJitter + time.Second)
```

Comment:
That's a lot of time to sleep in a single test. Let's make it configurable for the tests?

Reply:
Also if delete repo isn't using the cache then why wait?

Reply:
I don't think it's that bad to add this sleep to only one test. Feels like adding a configuration just for the sake of a single test is a far worse option

Reply:
As long as there is no way to control the cache time or enable/disable - will need to wait while working with GetRepository.
Can make DeleteRepository skip the cache mechanism and write the test using two DeleteRepository calls to remove the sleep here.